src/NzbDrone.Core.Test/Files/Indexers/Nyaa/Nyaa.xml

```diff
@@ -1,40 +1,66 @@
-<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+<rss xmlns:atom="http://www.w3.org/2005/Atom"
```

Comment:
As Taloth mentioned on Discord, add this as a new file instead of trumping the existing one.